### PR TITLE
Add unit tests for pvc quota

### PR DESCRIPTION
**Pull request by shawnhanx**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
Add unit tests for pvc quota
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/quota/v1/evaluator/core/persistent_volume_claims_test.go [hunk 1]
```diff
@@ -136,3 +137,47 @@ func TestPersistentVolumeClaimEvaluatorUsage(t *testing.T) {
...
+			want: []corev1.ResourceName{},
+		},
+	}
+	for testName, testCase := range testCases {
+		actual := evaluator.MatchingResources(testCase.items)
+
+		if !reflect.DeepEqual(testCase.want, actual) {
+			t.Errorf("%s expected:\n%v\n, actual:\n%v", testName, testCase.want, actual)
+		}
+
```
Comment (yliaog) [inline pkg/quota/v1/evaluator/core/persistent_volume_claims_test.go:181]: remove extra blank line?

**Comment by dims** [comment]
@shawnhanx there's a tiny feedback left for you, once that is fixed, we can get this in. can you please look?

**Comment by shawnhanx** [comment]
> @shawnhanx there's a tiny feedback left for you, once that is fixed, we can get this in. can you please look?

Thanks, I have done it.

**Comment by shawnhanx** [comment]
> @shawnhanx there's a tiny feedback left for you, once that is fixed, we can get this in. can you please look?

Could you merge this PR, Thx?

**Comment by yliaog** [comment]
/cc @dims 

for approval

**Pull request merged**